Tie xenoarch point output to glimmer

kotobdev opened the pull request:
## About the PR
Artifact research now has a multiplier to point output tied to glimmer level, from 1x at 0 glimmer to 2.5x at 1000 glimmer, linearly.

## Why / Balance
Xenoarcheology is comically weak compared to anomalies. Despite requiring constant attention and being both RNG and departmentally dependent, xenoarch produces barely any research compared to anomalies, which you can basically ignore once you set them up. This change helps both remedy that issue, and also provides [yet another](https://github.com/DeltaV-Station/Delta-v/pull/4082) risk/reward incentive for Epi to increase glimmer. Ironically, even at 1000 glimmer, I doubt xenoarch will be faster then two (hell, even just one) decent anomalies in advanced containers. But whatever.
<sub>i'd LOVE to gather hard data on xenoarch efficiency in rounds, but that seems hard</sub>

To be entirely honest, the issue with xenoarcheology is probably less "xenoarch is weak" and moreso "anomalies are absolutely busted". But I'll leave that to another PR to fix.

## Technical details
xaml comment ewwww
xenoarch rework seems to be More Then A Week Away so im making this while i can 

~~right now it's a linear scale, would maybe be a bit cooler as an [exponential ease in/out](https://www.desmos.com/calculator/to2pcf6kwz), so epi doesnt reap many benefits until they really ramp up glimmer. didnt bother implementing that but i can give it a go if direction likes that. it's just a lot of evil arithmetic~~ updated

## Media
~~0, 500, and 1000 glimmer respectively.~~ images outdated, ui remains the same
<img width="300" alt="image" src="https://github.com/user-attachments/assets/11f1ded5-c599-4393-925c-3add12c2eb7c" />
<img width="300" alt="image" src="https://github.com/user-attachments/assets/186b17cf-52fb-49ba-9b1d-92093ee01ca5" />
<img width="300" alt="image" src="https://github.com/user-attachments/assets/6eb3676d-2709-4600-8a16-d6e834fd8814" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelogs must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- tweak: Artifact research point output is now tied to glimmer level, becoming exponentially more efficient as glimmer increases.

kotobdev committed:
it work i think

kotobdev committed:
tweakops

kotobdev committed:
commentops

kotobdev committed:
newline ops

Proxysseia commented:
Direction review 24h

Proxysseia commented on Content.Server/Xenoarchaeology/Equipment/Systems/ArtifactAnalyzerSystem.cs:
```diff
@@ -525,5 +535,12 @@ private void OnPowerChanged(EntityUid uid, ActiveArtifactAnalyzerComponent activ
+    {
+        return 1 + (_glimmerSystem.Glimmer / 1000f) * comp.PointGlimmerMultiplier;
+    }
+    // End DeltaV - glimmer mult calculation
```
We like the direction of this PR and would like to double down. 
Exponential scaling tied to higher glimmer. We want high glimmer to also present an opportunity and to not just be a 'stop all work' state.
In the future we should look into adding more ways to manually increase or decrease glimmer, but that is outside the scope of this PR.

kotobdev committed:
direction + bugfix ops

exponential point increase, no more insane glimmer nukes (sad!), analyzer no longer displays negative points

kotobdev commented:
@deltanedas review plz :D
